docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -91,6 +93,30 @@ If you get frequent cluster alerts on a [Sandbox cluster](../../cloud-basics/cre
 
 {{< /note >}}
 
+#### Fix throughput alerts
+
+YugabyteDB Managed sends a notification when the disk throughput on any node in the cluster exceeds the threshold, as follows:
+
+- Node disk throughput utilization exceeded 80% (Warning).
+- Node disk throughput utilization reached 100% (Warning).
+
+If disk throughput consistently exceeds 80% for any node, consider taking action to remove any bottlenecks on performance. For clusters deployed on AWS, increase the maximum disk IOPS per node. Throughput is automatically scaled together with IOPS. For clusters deployed on GCP or Azure, add more vCPUs or increase the disk size.
+
+For information on scaling clusters, refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
+
+#### Fix IOPS alerts
+
+YugabyteDB Managed sends a notification when the disk input output (I/O) operations per second (IOPS) on any node in the cluster exceeds the threshold, as follows:
+
+- Node disk IOPS utilization exceeded 80% (Warning).
+- Node disk IOPS utilization reached 100% (Warning).
+
+For large datasets or clusters with high concurrent transactions, higher IOPS is recommended.
+
+If disk IOPS consistently exceeds 80% for any node, consider taking action to remove any bottlenecks on performance. For clusters deployed on AWS, increase the maximum disk IOPS per node. For clusters deployed in GCP and Azure, add more vCPUs or increase the disk size.
```

Comment:
```suggestion
If disk IOPS consistently exceeds 80% for any node, consider taking action to remove any bottlenecks on performance. For clusters deployed on AWS, increase the maximum disk IOPS per node. For clusters deployed in GCP and Azure, add more vCPUs and/or increase the disk size.
```

Reply:
This project is specific for AWS, the alerts won't be triggered for GCP/Azure clusters.

Reply:
GCP/Azure cluster don't have the concept of IOPS, should not expect alerts for them

Reply:
You can't _change_ IOPS for GCP/Azure, but you should still get alerts for them, no? Metrics should still be reported for them - @vmallepalli

Reply:
Agreed, IMO we should still alert for gcp/azure since it's useful information that can explain any performance bottlenecks a customer sees. And while they cannot explicitly modify iops to xxx amount, they can still increase their IOPS indirectly by scaling up the disk size and/or increasing their vcpus (this is documented by both gcp and azure). So it's still actionable and we can mention these options here if we want.

Reply:
So, are there IOPS alerts for GCP/Azure now, or would that have to be added? I was assuming that there are functioning IOPS alerts for GCP/Azure now. @vmallepalli

Reply:
I think the IOPS configuration and alert are only built for AWS only. Running following returns nothing
```
ybp_universe_node_provisioned_iops{cloud='gcp'}
ybp_universe_node_provisioned_iops{cloud='azure'}
```